src/pages/commits/CommitsWrapper.tsx

```diff
@@ -51,6 +52,12 @@ export const CommitsWrapper: React.FC<Props> = ({
   const updateQueryParams = useUpdateURLQueryParams();
   const parsed = parseQueryString(search);
 
+  useEffect(() => {
+    if (!isLoading) {
+      hoverTaskIcons();
+    }
+  }, [isLoading, versions]);
```

Comment:
I think I need to update on the `versions` because any 5 of the versions could change... but this seems a little excessive possibly